magpie/cli/purge_expired_network_tokens.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+"""
+Removes expired network tokens from the database.
```

Comment:
Seems like a useful operation that should be directly in the API (eg: `DELETE /network/tokens` or similar). 
The CLI could simply call that operation and provide the errors/success based on what the API responds.

Reply:
I agree that it's useful to have. 

I'll add it in the API but I'd like to keep the option to perform the deletion on the DB directly because that way we can simplify the crontab entry by not requiring a username and password.

Reply:
Added an option to do this with an API call